### ISSUE TITLE
Today widgets crashes SpringBoard (iOS 10)

### ISSUE THREAD

**Issue by albyvar25** [DGh0st/HSWidgets#16]
As title says. I tried disabling all tweaks except hswidgets but still crashes. Also tried version 0.3 (since changelog says fixed today widgets crash on ios 10) but it crashes too. Could you help me please?

**Comment by DGh0st** [comment]
Have you tried latest version?
Can you provide a crashlog?

**Comment by albyvar25** [comment]
Yeah, I tried latest version it has the same issue unfortunately. About the crash log: cr4shed and crashreporter won't show any info. All I was able to see was a syslog. Not sure if it can help (i've only included the part where a crash in springboard is visible. let me know if you need another kind of log
[1.log](https://github.com/DGh0st/HSWidgets/files/8054164/1.log)
)

**Comment by DGh0st** [comment]
There is no info about the crash in that log. The crashlog with backtrace would help, or syslog that contains a log of an exception (if there was an exception that caused the crash).

The only other thing I can suggest is to try getting a clean install of the tweak. That means removing all preference files and then reinstalling.

**Comment by Tuturmllss** [comment]
Hello, I am having the same issue on my ios 10 ipad. Deleting the preference files and reinstalling does not change anything unfortunately. Any other idea? Thanks

**Comment by DGh0st** [comment]
@Tuturmllss can you provide any crashlogs?

**Comment by Tuturmllss** [comment]
Hi @Dgh0st, thank you for your reply. Please find attached
[SpringBoard-2022-06-07-213903.zip](https://github.com/DGh0st/HSWidgets/files/8856250/SpringBoard-2022-06-07-213903.zip)

**Comment by DGh0st** [comment]
Sorry about the delay. Looks like the widget is crashing on its own.

Does this happen for all widgets?
Also have you tried version before the HSWidget rewrite (0.0.1-6)? It doesn't have a lot of the new features but just want to see if this crash was introduced recently or not.
